### PR TITLE
Increase initial delay for liveness probe test pod from 15s to 30s.

### PR DESCRIPTION
**Pull request by vishh**
For #18704

### CODE REVIEW

File: test/e2e/pods.go [hunk 1]
```diff
@@ -689,7 +689,7 @@ var _ = Describe("Pods", func() {
 									Port: intstr.FromInt(8080),
 								},
 							},
-							InitialDelaySeconds: 15,
+							InitialDelaySeconds: 30,
 							FailureThreshold:    1,
```
Comment (timstclair) [inline test/e2e/pods.go]: Alternatively, we could increase the `FailureThreshold`, although I think that's probably more likely to mask legitimate bugs.

Comment (vishh) [inline test/e2e/pods.go]: Yeah. I also considered setting `TimeoutSeconds`, but even that doesn't address the basic issue.

**Comment by timstclair** [comment]
LGTM

File: test/e2e/pods.go (hunk 1 above)
Comment (timstclair) [inline test/e2e/pods.go]: Yeah, just saw your comment on the issue. Lets see if setting the InitialDelaySeconds helps.

**Comment by fabioy** [comment]
Manually merging.

**Pull request merged**